README.md

```diff
@@ -230,6 +230,37 @@ Both subfields also need to have their `host` and `service` specified. If you de
 * `host` should be `memcached-app.loki.svc`. Otherwise, with custom values for `memcached-app`, the `host` value will be memcached's service DNS name.
 * `service` should be `memcache`. With custom values for `memcached-app`, the `service` value will be memcached's service port name.
 
+### Bloom filters
+
+Giant Swarm experimented with bloom filters quite early one after the release of Loki 3.1.0 as can be seen [here](https://github.com/giantswarm/roadmap/issues/3563).
+
+You can quite easily enable blooms in your loki instance by setting the following configuration:
+
+```yaml
+loki:
+  loki:
+    structuredConfig:
+      bloom_compactor:
+        enabled: false
```

Comment:
how do we enable it ?
There is `bloom_compactor.enabled` and also `limits_config.bloom_compactor_enable_compaction`.

Reply:
You need both. 
`bloom_compactor.enabled` enables the component
`limits_config.bloom_compactor_enable_compaction` enables the feature

Reply:
Ok, so I suggest to update the config example then.